packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -4127,6 +4130,7 @@ sealed class _SemanticsBase extends SingleChildRenderObjectWidget {
            role: role,
            controlsNodes: controlsNodes,
            validationResult: validationResult,
+           hitTestBehavior: hitTestBehavior,
```

Comment:
we may need to force node that sets this to not merge up

For example
```dart
Column(
  children: [
     Text('child1'),
     Semantics(hitTestBehavior: .opaque, child: Text('child2')),
     Text('child3'),

  ]
)
```

If you print the semantics tree you will see the entire column will form a node, and the opaque will merge up to apply to the entire node

There are two way to fix this
1. force Semantics widget that sets hitTestBehavior to also have container:true
2. implement SemanticsConfiguration.isCompatible to reject the merge in this case

I think going with (2) makes more sense